src/api/species/species.ts

```diff
@@ -97,6 +97,7 @@ type SimpleSuccessResponsePayload =
  */
 export type UpdateSpeciesResponse = {
   requestSucceeded: boolean;
+  responseMessage?: string;
```

Comment:
this is fine... here's an example of how we have been naming the property recently 
https://github.com/terraware/terraware-web/blob/main/src/api/accessions2/accession.ts#L27
only reason we added a util is to remove some boilerplate code